state/controller.go

```diff
@@ -144,12 +143,9 @@ func (st *State) checkValidControllerConfig(updateAttrs map[string]interface{},
 			return errors.Trace(err)
 		}
 
-		if k == jujucontroller.JujuHASpace || k == jujucontroller.JujuManagementSpace {
-			cVal := updateAttrs[k].(string)
-			if err := st.checkSpaceIsAvailableToAllControllers(cVal); err != nil {
-				return errors.Annotatef(err, "invalid config %q=%q", k, cVal)
-			}
-		}
+		// TODO(nvinuesa): When we complete the migration to dqlite
```

Comment:
This validation will be handled by the charm. It's OK to leave this TODO, but this whole area changes completely.